app/src/ui/clone-repository/clone-repository.tsx

```diff
@@ -97,6 +102,8 @@ export class CloneRepository extends React.Component<
     if (initialURL) {
       this.updateUrl(initialURL)
     }
+
+    window.addEventListener('focus', this.onFocus)
```

Comment:
Could we move this to `componentDidMount` and also add remove the listener in `componentWillUnmount` to avoid a dangling subscription and subsequent memory leak? See `dialog.tsx` for an example of how we've done this previously.